### PR TITLE
Final V1.0 Specification

### PR DESCRIPTION
**Pull request by apinilabs-daniel**
Please review the Final V1.0 Specification of the `PAC-ID`. Let me know when ready to merge or make and comment any changes you like to be done first.

### CODE REVIEW

File: well-known-id-segment-keys.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Well-known `id segment key`s
+
+It is RECOMMENDED to use well-known `id segment key`s.
+
+The following approach SHALL be used to define an `id segment key`:
+
+1. Use a preexisting [GS1 Application Identifier](https://ref.gs1.org/ai/). Some commonly used GS1 AIs are referenced in the table below.
+2. Use a predefined key from the table below (with origin `PAC-ID-COMMONS`).
+3. Use a self-defined key. It is RECOMMENDED to base such a key on a well-known existing ontology. New keys MAY be added to the table below through a merge request.
```
Comment (apinilabs-pascal) [inline well-known-id-segment-keys.md]: merge request is named pulled request at GitHub.

File: README.md [hunk 2]
```diff
@@ -1,140 +1,158 @@
...
+| `issuer` | The party which issued the identifier and knows what the identifier refers to. | <ul><li>MUST be a valid domain name according to [RFC 1035](https://www.ietf.org/rfc/rfc1035.html).</li><li>SHOULD be a registered and active domain name.</li><li>SHOULD contain only the following characters `A-Z`, `0-9`, `-`, and `.`</li></ul> | "METTORIUS.COM"<br>(The manufacturer of the balance) |
+| `identifier` | The identifier itself. | <ul><li>MUST consist of one or more `id segments` separated by `/`.</li><li>At least one `id segment` MUST be non-empty.</li><li>MUST not exceed 256 characters.</li></ul> | "DEVICE/21:210263"<br>(An identifier for one particular balance) |
+| `id segment` | The `id segment` is a part of an `identifier` that can stand on its own. Typically used to organize `identifier`s within an `issuer`. | <ul><li>MUST be a valid `hsegment` according to [RFC 1738](https://www.ietf.org/rfc/rfc1738.txt), but without `*` (see **PAC-ID Extension**). </li><li>SHOULD be limited to `A-Z`, `0-9`, and `:-+` for new designs.</li><li>CAN be an `id segment key` and `id segment value` pair separated by `:`.</li></ul> | "21:210263"<br>(A id segment containing a serial number) |
```
Comment (apinilabs-pascal) [inline README.md]: typo in segment.

File: README.md [hunk 3]
```diff
@@ -1,140 +1,158 @@
...
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt) "Key words for use in RFCs to Indicate Requirement Levels".
 
-**A:** Not by design. In principle PAC-IDs could be used everywhere where real world objects need to be unambiguously identified and referred to among otherwise unrelated systems.
+## FAQ
 
-**Q: Is there any recommendation for the identifier?**
+**Q: Why is a PAC-ID represented as an HTTPS URL?**
 
-**A:** Basically the `identifier` is almost free to use with any format you'd like. Also, you can just reuse your existing identifiers. Other than that it's recommended to use a 128-bit UUID as identifier (encoded as base64url according to [RFC 4648](https://www.ietf.org/rfc/rfc4648.html)).
+**A:** The biggest advantage is that PAC-IDs can directly be entered as address into a browser (or scanned via QR code for example) on almost any device. To be able to easily recognize a PAC-ID as such, it was decided to use the subdomain `PAC.` instead.
```
Comment (apinilabs-pascal) [inline README.md]: "instead" can probably removed. otherwise make more clear that we could have used PAC:// instead

File: README.md [hunk 4]
```diff
@@ -144,14 +162,6 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
...
-
-**Q: How do I identify a well or multiple wells of a microtiter plate (MTP) (which has a PAC-ID itself) by a PAC-ID?**
-
-**A:** MTP well identifiers are not specified in order to keep PAC-IDs simple. If you like to identify wells by coordinates, it is recommended to append it to the identifier with "." as separator like in this example: `MDEyMzQ1Njc4OTAxMjM0NQ.H7`. Unfortunately, there is no standardized naming of wells, however the following scheme is widely used: 96-well plate coordinates range from `A1` to `H12`, 384-well plates coordinates range from `A1` to `P24`, 1536-well plates coordinates range from `A1` to `AF48`, ... Multiple wells (`E8;A5`) and well ranges (`A1:H1`) or combinations (`E8:H8;A9:D9`) can be addressed using "Excel" cell addressing style. Do not forget to URL encode the query string, like in this example for addressing well range `E8:H8;A9:D9`: `https://pac.apinilabs.com/myapp?i=MDEyMzQ1Njc4OTAxMjM0NQ.E8%3AH8%3BA9%3AD9`
-
 **Q: I like that a PAC-ID points to my own (proprietary or local) service instances, e.g. to access my own ELN, LIMS, inventory system, ...**
```
Comment (apinilabs-pascal) [inline README.md]: to have a question add something like "How to achieve that?" at the end.

File: README.md (hunk 2 above)
Comment (apinilabs-daniel) [reply README.md]: Cannot find the typo @apinilabs-pascal . if you meant "hsemgent", than this is correct, it refers to RFC 1738. If you meant something else, please correct directly in V1.0 branch.

**Pull request merged**